page-faq.php

```diff
@@ -68,6 +68,9 @@
 <p>Many people prefer to have more control over who gets to see the photos they send to their spouse or friends, who gets to read their bank account statements or determine their political allegiance. Or know where they are - as mobile phones track your location pretty much continuously...</p>
 <p>This is where ownCloud and similar technologies come in. ownCloud offers you the option to take back your data. Once you install it on a server (or let somebody do that for you! <a href="https://owncloud.org/blog/dont-be-caught-naked-in-the-cloud-decentralization-protects-our-data/" target="_blank">Decentralization is an important tool to protect privacy</a>) you can access your private data in an easy to use web interface or synchronize it with your devices like Android and iPhones. You can store your contacts and calendar as well as files on ownCloud and, through the many ownCloud apps, use it to store passwords, play music and movies and so on. It also allows you to securely share with other people and collaborate on documents. Learn more <a href="/features">about ownCloud features here</a>.</p>
 <p>All this is free, as ownCloud is available under an '<em>open source license</em>', which gives you the right to examine, share <strong>and</strong> modify it. ownCloud is developed by an <a href="/contact">international community</a> of both paid and volunteer contributors and <a href="/contribute">you can get involved, too</a>.</p>
+<h3>Things ownCloud does not do</h3>
+<p>Note that ownCloud is NOT a backup tool: the desktop client, for example, will synchronize the contents of a local folder with the server. That works for file modifications and additions, but also deletions. If you delete a file on the server, it gets deleted on the desktop you installed the client on! ownCloud does have a <strong>trasn</strong> and <strong>Versions</strong> support, which means deleted or modified files can be brought back, but this can be disabled and depending on the space usage settings, these older files do get deleted over time. You should always backup your ownCloud server!</p>
```

Comment:
In addition, no need to say "a Trash and Versions support" - just "Trash and Versions support".